docs/content/doc/usage/actions/act-runner.en-us.md

```diff
@@ -172,6 +172,29 @@ It is because the act runner will run jobs in docker containers, so it needs to
 As mentioned, you can remove it if you want to run jobs in the host directly.
 To be clear, the "host" actually means the container which is running the act runner now, instead of the host machine.
 
+### Configuring cache when starting a Runner with Docker image
+When starting a runner using a Docker image, it is essential to configure the cache action to ensure its proper functioning. Follow these steps:
```

Comment:
It should be clear why we need to do this or what it is for. And it's still optional because not everyone needs `actions/cache`.